templates/react/components/foo/List.js

```diff
@@ -84,4 +77,13 @@ const mapDispatchToProps = (dispatch) => {
   };
 };
 
+List.propTypes = {
```

Comment:
Yes I agree for `React.PropTypes`, but I was asking why moving the definition from a `static` property in `List` to an external call to `List.propTypes`?